ui/app/adapters/pki/tidy.js

```diff
@@ -13,34 +12,32 @@ export default class PkiTidyAdapter extends ApplicationAdapter {
     return `${this.buildURL()}/${encodePath(backend)}`;
   }
 
-  urlForCreateRecord(snapshot) {
```

Comment:
opted for custom `createRecord` method so that the UI renders the warning returned as a `FlashMessage`